### PR TITLE
types shall always be plural

### LINKED ISSUE #22052: Deleted user not returned properly in comments Webdav response
### Steps

1. Create a user "user1"
1. Create a file "test.txt"
1. Post a few comments as "user1" on "test.txt"
1. Share "test.txt" with the admin
1. Login as admin
1. Delete the user "user1"
1. Check comments on "test.txt" (webdav API)

### Expected result
Webdav API for REPORT returns "actorType" as "deleted_user".

### Actual result
Webdav API for REPORT returns "actorType" as "users".

@blizzz as discussed

### PR DESCRIPTION
**Pull request by blizzz**
Should fix https://github.com/owncloud/core/issues/22052

@PVince81 please note, the deleted user type is now also harmonized to plural

### CODE REVIEW

File: lib/public/comments/icomment.php [hunk 1]
```diff
@@ -161,7 +161,7 @@ public function getActorId();
 	/**
 	 * sets (overwrites) the actor type and id
 	 *
-	 * @param string $actorType e.g. 'user'
+	 * @param string $actorType e.g. 'users'
```
Comment (PVince81) [inline lib/public/comments/icomment.php]: I didn't realize I've been sending the wrong value all along :confused: 

Maybe the Webdav backend should validate the type and only allow "users" for now ?

**Comment by PVince81** [comment]
:+1: now for a deleted user, the REPORT call properly returns the actor type "deleted_users".

I'll adjust the UI later to react on that.

File: lib/public/comments/icomment.php (hunk 1 above)
Comment (blizzz) [inline lib/public/comments/icomment.php]: I've been mixing this up myself :speak_no_evil:

**Comment by MorrisJobke** [comment]
Tested as worked :+1:

**Pull request merged**